### DISCUSSION TITLE
How I can run this project.

### DISCUSSION THREAD

**Discussion by MQuashif** [lllyasviel/Omost#50] [Q&A]
I am not able to run this project. Everything is done but not able to run or open GUI of this project

**Comment by xhoxye** [comment]
The documentation has installation instructions, 
git clone 
1. Install conda, 
2. Create a virtual environment, 
3. Activate the virtual environment, 
4. Install dependencies in the virtual environment, 
5. Start the gradio_app.py in the virtual environment.
`git clone https://github.com/lllyasviel/Omost.git`
`cd Omost`
`conda create -n omost python=3.10`
`conda activate omost`
`pip install torch torchvision --index-url https://download.pytorch.org/whl/cu121`
`pip install -r requirements.txt`
`python gradio_app.py`
